test/cadet/public/updater_test.exs

```diff
@@ -31,4 +34,71 @@ defmodule Cadet.Public.UpdaterTest do
       assert String.length(token) == 480
     end
   end
+
+  test "Get course id" do
+    use_cassette "updater/get_course_id#1", custom: true do
+      course_id = Updater.get_course_id("T0K3N...")
+      assert String.length(course_id) == 36
+      assert course_id != "00000000-0000-0000-0000-000000000000"
+    end
+  end
+
+  test "Get announcements" do
+    use_cassette "updater/get_announcements#1", custom: true do
+      token = Updater.get_token()
+      course_id = Updater.get_course_id(token)
+      assert {:ok, announcements} = Updater.get_announcements(token, course_id)
+      assert is_list(announcements)
+    end
+  end
+
+  describe "Start GenServer" do
+    test "Using GenServer.start_link/3" do
+      use_cassette "updater/start_link#1", custom: true do
+        assert {:ok, _} = GenServer.start_link(Updater, %{}, name: TestUpdater)
+        assert :ok = GenServer.stop(TestUpdater)
+      end
+    end
+
+    test "Using Updater.start_link/1" do
+      use_cassette "updater/start_link#2", custom: true do
+        assert {:ok, pid} = Updater.start_link()
+        assert Process.alive?(pid) == true
```

Comment:
`== true` is superfluous, can be removed

Reply:
Done with a2ae3d9.